Adjust quickstart to not show commands example (#6793)
Rapptz/discord.py: docs/quickstart.rst

        if message.author == client.user:
            return

>         if message.content == ('Hello World!'):

Reviewer: ```suggestion
        if message.content == 'Hello World!':
```

>         if message.content.startswith('hello'):

Reviewer: ```suggestion
        if message.content.startswith('$hello'):
```

4. Since the :func:`on_message` event triggers for *every* message received, we have to make
   sure that we ignore messages from ourselves. We do this by checking if the :attr:`Message.author`
   is the same as the :attr:`Client.user`.

> 5. Afterwards, we check if the :class:`Message.content` starts with ``'hello'``. If it does,

Reviewer: ```suggestion
5. Afterwards, we check if the :class:`Message.content` starts with ``'$hello'``. If it does,
```

> 5. Afterwards, we check if the :class:`Message.content` is ``'Hello World!'``. If it is,
>    then we send a message in the channel it was used in with ``'Hello!'``. This is a basic way of 
>    handling commands, which can be later automated with the :ref:`ext.commands` framework.

Reviewer: ```suggestion
5. Afterwards, we check if the :class:`Message.content` is ``'Hello World!'``. If it is,
   then we send a message in the channel it was used in with ``'Hello!'``. 
```

Author: I strongly think that a reference to ext.commands should be included somewhere on this page, if not here then somewhere else.
